microcosm_postgres/store.py

```diff
@@ -199,10 +199,10 @@ def _order_by(self, query, **kwargs):
         """
         Add an order by clause to a (search) query.
 
-        By default, is a noop.
+        By default, is by created_at.
 
         """
-        return query
+        return query.order_by(self.model_class.created_at.desc())
```

Comment:
we instead use: 
```
        if hasattr(self.model_class, "created_at"):
            return query.order_by(self.model_class.created_at.desc())
        return query
```
but both `EntityMixin` and `UnixTimestampEntityMixin` are based on `PrimaryKeyMixin` - so it seems safe enough

Reply:
Do we have indexes on these criteria by default?

Reply:
No. only on `id`